button.css

```diff
@@ -0,0 +1,103 @@
+.btn-container{
+    width: 525px;
+    height: 91px;
+    display: flex;
+    gap: 40px;
+    flex-wrap: wrap;
+}
+
+.btn-container .primay-btn{
+    padding: 9px 10px;
+    margin: 20px 20px;
+    border: none;
+    letter-spacing: 1px;
+    border-radius: 6px;
+}
+
+.label-btn{
+    background-color: var(--primary-color);
+    width: 90px;
+    height: 37px;
+    color: var(--font-color);
+}
+
+/* button with outline */
+
+.btn-container .outl-btn{
+    background-color:var(--secondary-color);
+    border:2px solid var(--primary-color);
+    color:var(--primary-color);
+    width:100px;
+    height:40px;
+    transition:all 0.5s ease-in-out;
+}
+
+.btn-container .outl-btn:hover{
+    cursor: pointer;
+    background-color:var(--primary-color);
+    color:var(--font-color);
+}
+
+/* button with icon */
+
+.btn-container .icon-btn{
+    width: 107px;
+    height: 40px;
+    display: flex;
+    color:var(--primary-color);
+    justify-content: space-between;
+    background: linear-gradient(
+110deg
+, #124490 35%, white 36%);
+font-weight:bold;
+}
+
+.btn-container .icon-btn i{
+    font-size:17px;
+    color:white;
+}
+
+.btn-container .sec-btn{
```

Comment:
see if having classes like `btn-secondary` or `btn-primary` sound better? and it will follow the BEM standard as well.